src/mlpack/methods/hmm/hmm_viterbi_main.cpp

```diff
@@ -72,6 +73,10 @@ int main(int argc, char** argv)
   // Parse command line options.
   CLI::ParseCommandLine(argc, argv);
 
+  if (CLI::HasParam("output_file"))
+    Log::Warn << "--output_file (-o) is not specified;"
+      << "no results will be saved!" << endl;
```

Comment:
Could you reformat this a little bit please? (There are lots of messages like this so my comment applies to all of them, not just this one.)  More of the message might fit on one line, and when you wrap, be sure to have two extra indents before the `<< "rest of the message." << endl` part.